pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
@liggitt IOW. you will have to write "bash -c rake test" which we want to avoid to make this more friendly for developers. Which is not ideal, but we can't make it clean until SCL provide us with already enabled image without bash wrappers